sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/QueryEventsRequestOptions.cs

```diff
@@ -16,6 +16,12 @@ public class QueryEventsRequestOptions : QueryRequestOptions
         /// </summary>
         public List<EventProperty> ProjectedProperties { get; }
 
+        /// <summary>
+        /// The maximum number of property values in the whole response set, not the maximum number of property values per page.
+        /// Defaults to 10,000 when not set. Maximum value of take can be 250,000.
+        /// </summary>
+        public int? MaximumNumberOfEvents { get; set; }
```

Comment:
Just wondering why you chose Events vs Properties?

Reply:
Good point. Will change.

Reply:
actually, maybe Events makes sense, given that this class is for QueryEventsRequestOptions.cs. 

It's not properties because it's not the maximum number of properties returned in the response (ie, Temperature, Humidity, etc..), rather number of events.

Reply:
What is an example of event here if not property (just for my understanding)?